src/ontology/mondo-edit.obo

```diff
@@ -262406,13 +262410,15 @@ xref: ICD9:277.85 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: MESH:C566945 {source="Orphanet:746", source="MONDO:equivalentTo", source="Orphanet:746/e", source="https://github.com/monarch-initiative/mondo/issues/2212"}
 xref: NCIT:C98991 {source="MONDO:equivalentTo"}
 xref: OMIM:609015 {source="Orphanet:746", source="MONDO:equivalentTo", source="Orphanet:746/e"}
+xref: OMIMPS:609015 {source="MONDO:equivalentTo"}
```

Comment:
this cannot be equiv to a proper OMIM term and an OMIMPS

Reply:
This one was missed.

Reply:
updated according to my logic above.